Sources/MediaNavigationBar.swift

```diff
@@ -77,59 +88,41 @@ protocol MediaNavigationBarDelegate {
     override init(frame: CGRect) {
         super.init(frame: frame)
         setupViews()
-        setupConstraints()
     }
     
     // MARK: Instance Methods
     func setMediaTitleLabelText(_ titleText: String) {
         mediaTitleTextLabel.text = titleText
     }
     
-    private func setupConstraints() {
-        let margin: CGFloat = 35.0
-        var airplayView: UIView
-        
-        if #available(iOS 11.0, *) {
-            airplayView = airplayRoutePickerView
-        } else {
-            airplayView = airplayVolumeView
-        }
-        
-        let constraints: [NSLayoutConstraint] = [
-            minimizePlaybackButton.leadingAnchor.constraint(equalTo: leadingAnchor),
-            minimizePlaybackButton.topAnchor.constraint(equalTo: topAnchor),
-            minimizePlaybackButton.bottomAnchor.constraint(equalTo: bottomAnchor),
-            mediaTitleTextLabel.leadingAnchor.constraint(equalTo: minimizePlaybackButton.trailingAnchor, constant: margin),
-            mediaTitleTextLabel.topAnchor.constraint(equalTo: topAnchor),
-            mediaTitleTextLabel.trailingAnchor.constraint(lessThanOrEqualTo: chromeCastButton.leadingAnchor, constant: margin),
-            chromeCastButton.topAnchor.constraint(equalTo: topAnchor),
-            chromeCastButton.trailingAnchor.constraint(equalTo: airplayView.leadingAnchor, constant: margin),
-            airplayView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: margin),
-            airplayView.topAnchor.constraint(equalTo: topAnchor)
-        ]
-        
-        NSLayoutConstraint.activate(constraints)
-    }
-    
     private func setupViews() {
-        addSubview(minimizePlaybackButton)
-        addSubview(mediaTitleTextLabel)
-        addSubview(chromeCastButton)
+        axis = .horizontal
```

Comment:
`UILayoutConstraintAxisHorizontal` should already be the default on UIStackViews.